docs/development_suite/identity-and-access-management/manage-groups.md

```diff
@@ -28,19 +30,19 @@ At Company level, for each group, it is possible to manage:
 * Role: the group Company role
 * Members: adding and/or removing users from the group
 
-Existing groups can also be found within the IAM portal at both Company and Project level, where it is possible to manage roles assigned to them just like for the other identity types. 
+You can also locate existing groups in the IAM portal, both at the Company and Project levels. In these sections, you have the ability to handle the roles assigned to these groups, similar to how you manage roles for other types of identities.
 
-Groups allow you to efficiently manage accesses to your users. Nevertheless, they are not the only possible way to manage permissions on a Company. As a matter of fact, you can still manage user privileges at individual level, or even handle them through a combination of both methods. If you want to discover more about user management, visit the dedicated [documentation page](/development_suite/identity-and-access-management/manage-users.md#how-to-best-manage-your-users).
+:::info
```

Comment:
does an info followed by a caution appear good? isn't it a bit too much impact for the eye? 🤔

Reply:
I agree with you, the two blocks being so close together are not visually appealing. I combined this "info" block with the one at the top of the section, as the context was more or less the same.